### PR TITLE
Rename Presto references in OAuth2 success.html

### PR DESCRIPTION
**Pull request by hovaesco**
I have noticed Presto references for OAuth2 authentication succeeded window.
image

### CODE REVIEW

File: core/trino-main/src/main/resources/oauth2/success.html [hunk 1]
```diff
@@ -44,7 +44,7 @@
     <link href="/ui/vendor/css-loaders/loader.css" rel="stylesheet">
 
     <!-- Custom CSS -->
-    <link href="/ui/assets/presto.css" rel="stylesheet">
+    <link href="/ui/assets/trino.css" rel="stylesheet">
```
Comment (kokosing) [inline core/trino-main/src/main/resources/oauth2/success.html:47]: don't you need to rename some file?

**Review by kokosing — COMMENTED** [review]
Can you please share the screen shoot how it looks after the change?

File: core/trino-main/src/main/resources/oauth2/success.html (hunk 1 above)
Comment (hovaesco) [reply core/trino-main/src/main/resources/oauth2/success.html:47]: there is no `presto.css` file, `trino.css` is already used in various HTMLs like oauth2 `failure.html`

**Comment by hovaesco** [comment]
Before:
image

After:
image

File: core/trino-main/src/main/resources/oauth2/success.html (hunk 1 above)
Comment (kokosing) [reply core/trino-main/src/main/resources/oauth2/success.html:47]: Oh... so it is actual separate issue then. One is to change the content and other one is to fix the referenced file for css. Please do two commits for this.

**Review by kokosing — APPROVED** [review]
% comment

**Review by Praveen2112 — APPROVED** [review]

**Comment by kokosing** [comment]
CI hit: https://github.com/trinodb/trino/issues/9444

**Pull request merged**